changelogs/unreleased/3198-shellwedance

```diff
@@ -0,0 +1 @@
+Added an if conditional that checks the length of `pod.Status.InitContainerStatus` to prevent potential error.
```

Comment:
Can you reword this message to be less about the code and more about the user impact? What's the potential error that's being fixed that users should know about?

Reply:
Right it seems code-friendly rather than user. 
Modified.